README.md

```diff
@@ -38,19 +38,29 @@ Of course, portcos can and will differ — we expect the final call on everythin
   * AWS Fargate after that.
   * Raw EC2 if nothing else works.
 * Frontend:
+  * HTML5
   * CSS: Tailwind
-  * JavaScript: 
+  * JavaScript:
+    * ES6 is recommended - https://kangax.github.io/compat-table/es6/
+    * [TypeScript](https://www.typescriptlang.org/) is also nice
     * no framework ("vanilla js") until that doesn't work
-    * Vue.js after that
+    * [Vue.js](https://vuejs.org/) or [Ractive.js](https://ractive.js.org/) after that
   * Asset management: Parcel
-* GIS: 
+* GIS:
   * Database: PostGIS
   * Backend framework: Django
   * Slippy maps: OpenLayers
 * Infrastructure Management: Terraform
-* Job queue: 
+* Job queue:
   * Celery + Redis
   * arteria/django-background-tasks for prototypes/demos, where the added cost/complexity of a real job queue isn’t worth the effort.
-* Search: 
+* Search:
   * Postgres full-text for commodity search
-  * ElasticSearch where search is a core component, or where Postgres fails
+  * Elasticsearch where search is a core component, or where Postgres fails
+* Metrics:
```

Comment:
So the rest of this is lovely and thank you! My only hesitation is here -- I've not used either of these, so it's a bit hard for me to tell if my preference (ELK) is _really_ a better default, or if I'm just being grouchy here. @vzvenyach @nvembar can you gut-check me here?

Reply:
Nurries & feel free to drop them. I've not used ELK & I don't have a strong preference. More added so people are thinking about it than anything else.

Reply:
After thinking a bit more - I think I'm just gonna go ahead and merge this as-is; nothing here is set in stone, so if we're wrong then we're wrong and we fix it later :)